src/utils/exr-parser.worker.js

```diff
@@ -17,102 +17,104 @@
  *  };
  */
 
-let openEXRLoaded = false;
-let queuedJobs = [];
-let OpenEXR;
+exports.__esModule = true;
 
-self.addEventListener('message', (event) => {
+var EXR = require("../exr-wrap/exr-wrap.js");
+
+// eslint-disable-next-line no-restricted-globals
+var ctx = self;
+var openEXRLoaded = false;
+var queuedJobs = [];
+var openEXR;
+
+ctx.addEventListener('message', function (event) {
     if (!openEXRLoaded) {
         queuedJobs.push(event.data);
-    } else {
+    }
+    else {
         handleJob(event.data);
     }
 });
 
-
-const exrwrapPath = require('file-loader?name=exr-wrap.js!../exr-wrap/exr-wrap.js');
-const exrwrapWasmPath = require('file-loader?name=exr-wrap.wasm!../exr-wrap/exr-wrap.wasm');
```

Comment:
This part doesn't just work when compiling this as an npm module, at least not in my env.